capstone/src/main/webapp/rider.html

```diff
@@ -12,14 +12,11 @@
 <body>
   <form action="/rider" method="POST" id="comment-adder" >
     <p>Enter rider info</p>
-    <input type-"text" name="rider-input">rider</input>
+    <input type-"text" name="rider-input"></input>
```

Comment:
Should be type="text"

Reply:
Improved will include in next pull request!